app/components/layer-settings/component.js

```diff
@@ -49,6 +49,15 @@ export default Ember.Component.extend({
         let offset = el.offset();
         $('body').animate({scrollTop:offset.top}, '500');
     },
+    didRender(){
+        let layerContent = this.get('layer.content.component');
+        if(layerContent === 'pages-menu' || layerContent === 'layer-link' || layerContent === 'layer-image-text' || layerContent === 'layer-advanced' || layerContent === 'layer-image'){
```

Comment:
This guy needs some spaces

Reply:
Check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Check on browser support though, otherwise you could do https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find and see if it doesn't return undefined

Reply:
you couls do `this.set("OSFContent", ["blocka", "blockb"].find(block => block === layerContent) === undefined)` or something